### ISSUE TITLE
Wait on Django RESt Framework ManyToManyField fix

### ISSUE THREAD

**Issue by normanjaeckel** [OpenSlides/OpenSlides#1515]
We do not want to have many to many fields to be required by default or if we use blank=True. This problem appears for tags and attachments in motion, tags in assignments and the like.

See https://github.com/tomchristie/django-rest-framework/pull/2674 and https://github.com/tomchristie/django-rest-framework/issues/2804.

**Comment by normanjaeckel** [comment]
See https://github.com/tomchristie/django-rest-framework/issues/3210. The fix is merge into master now and tagged with 3.2.0 release. So we can wait on this and update our deps. That's all.

**Comment by ostcar** [comment]
fixed
